prisma/schema.prisma

```diff
@@ -0,0 +1,73 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = "postgres://fndnziax:YEbewgjpbkceCXlmN46uXIYFuMPrkpbs@tyke.db.elephantsql.com/fndnziax"
```

Comment:
This URL contains the database password and should not be made public.

Reply:
URL added to .env file